### DISCUSSION TITLE
[Suggestion] Ignore Cloaked Teammates Only

### DISCUSSION THREAD

**Discussion by Eniac763** [Fedoraware/Fedoraware#981] [Ideas]
Under Visuals -> Players -> Player ESP -> Ignored Cloaked 
add teammates only.

The reason for this is because when enemy spy is cloaked and a teammate is also cloaked, it can be hard to decern which team they're on.

Example: ![20221114191226_1](https://user-images.githubusercontent.com/95100740/201794961-389f0193-033d-4723-9477-e94969eba5f1.jpg)

**Comment by SeasonalFerret** [comment]
I thought nametags were colored based on team?
Also, since you're using box esp (for health), why not enable outlines as well for enemies only?

**Comment by johnathon-walnut** [comment]
There's an option to change the colour to whatever you want - I assume this is what he's done

**Comment by SeasonalFerret** [comment]
why would you want esp on your teammates lmao it's just clutter
if anything, I use chams to make teammates 100% invisible

**Comment by Eniac763** [comment]
> I thought nametags were colored based on team? Also, since you're using box esp (for health), why not enable outlines as well for enemies only?

Nametags are colored based on team, however, when someone turns invisible they change to the invisible color which mine is currently gray. I also have glow ESP enabled for Enemy only, but when they turn invisible, the glow esp goes away.
